indexer/migrations/20230523_create_schema.sql

```diff
@@ -0,0 +1,99 @@
+/**
+ * BLOCK DATA
+ */
+
+CREATE TABLE IF NOT EXISTS l1_block_headers (
+	hash        VARCHAR NOT NULL PRIMARY KEY,
+	parent_hash VARCHAR NOT NULL,
+	number      NUMERIC NOT NULL,
+	timestamp   INTEGER NOT NULL,
+
+    CONSTRAINT
+        l1_parent_hash_fkey FOREIGN KEY(parent_hash) REFERENCES l1_block_headers(hash)
+);
+
+CREATE TABLE IF NOT EXISTS l2_block_headers (
+    -- Block header
+	hash                     VARCHAR NOT NULL PRIMARY KEY,
+	parent_hash              VARCHAR NOT NULL,
+	number                   NUMERIC NOT NULL,
+	timestamp                INTEGER NOT NULL,
+
+    -- Finalization information
+    l1_block_hash            VARCHAR NOT NULL REFERENCES l1_block_headers(hash),
+    legacy_state_batch_index INTEGER,
```

Comment:
These two are the two things missing making it impossible to figure out tx status correctly